packages/components/package.json

```
@@ -53,6 +53,7 @@
 		"@wordpress/primitives": "file:../primitives",
 		"@wordpress/rich-text": "file:../rich-text",
 		"@wordpress/warning": "file:../warning",
+		"ariakit": "^2.0.0-next.30",
```

Just remember to remove `^` if this is merged before the stable release or if it ends up using any unstable feature after that (to avoid this: https://github.com/ariakit/ariakit/pull/667#issuecomment-647108385).
```suggestion
		"ariakit": "2.0.0-next.30",
```

---

Good point! I don't think this PR will be ready anytime soon.

Do you already have a rough estimate of when `v2` (stable) will be released?

---

Pinned the `ariakit` version to `2.0.0-next.36` in the latest rebase.

> Do you already have a rough estimate of when v2 (stable) will be released?

Do you have any estimates around the v2 release date, @diegohaz ? And do you know if the `ariakit/select` can be considered stable at this point, API-wise?

---

I'm working to release it in October/November. But these estimates in software are complicated, also because it doesn't depend only on me.

Ultimately, people trying it out in their projects is what will make the library stable. When using a component, any positive or negative feedback is valuable.

The Select, specifically, has been a pretty popular component with positive feedback since its release, but it's also relatively new.